### PR TITLE
Fix the connection timeout with dead webhook endpoints

### PR DESCRIPTION
**Pull request by Fabio1988**
tried to fix the connection timeout with dead webhook endpoints

## Description
default timeout is now 30 seconds instead of 300. Can be configured in [docker-compose.yml](docker-compose.yml)

## Motivation and Context
related to issue #177 

## How Has This Been Tested?
Not tested.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Sources/RealDeviceMap/Controller/WebHookController.swift [hunk 1]
```diff
@@ -14,13 +14,18 @@ import PerfectCURL
 
 class WebHookController {
 
-    private init() {}
+    private init() {
+        let enviroment = ProcessInfo.processInfo.environment
+        timeout = enviroment["WEBHOOK_ENDPOINT_CONNECT_TIMEOUT"] ?? 30
```
Comment (123FLO321) [inline Sources/RealDeviceMap/Controller/WebHookController.swift:19]: needs to be parsed as Int
```swift
 timeout = enviroment["WEBHOOK_ENDPOINT_CONNECT_TIMEOUT"]?.toInt() ?? 30
 ```

**Review by 123FLO321 — CHANGES_REQUESTED** [review]

**Comment by Fabio1988** [comment]
have someone tested yet?

File: Sources/RealDeviceMap/Controller/WebHookController.swift [hunk 2]
```diff
@@ -238,7 +245,9 @@ class WebHookController {
             .addHeader(.contentType, "application/json"),
             .addHeader(.accept, "application/json"),
             .addHeader(.cacheControl, "no-cache"),
-            .addHeader(.userAgent, "RealDeviceMap \(VersionManager.global.version)")
+            .addHeader(.userAgent, "RealDeviceMap \(VersionManager.global.version)"),
```
Comment (tomhatzer) [inline Sources/RealDeviceMap/Controller/WebHookController.swift:248]: @Fabio1988 do you need the `\` here or can it be removed? (or is there one missing at the end?) 😃

**Comment by Fabio1988** [comment]
Be careful @123FLO321 ... these changes are also in #204

**Pull request merged**